src/fann.c

```diff
@@ -25,7 +25,7 @@
 #include <math.h>
 
 #include "config.h"
-#include "fann.h"
+#include "fann/fann.h"
```

Comment:
Wouldn't be better to use included path instead of prefixing all includes in the source?